README.md

```diff
@@ -1,94 +1,61 @@
 # LearnHub.mk Backend API
 
-#### Laravel Sail (Docker) installation
```

Comment:
Please split the installation instructions like it was done before so that it's not confusing to people. Both installation instructions are different so no need to use the DRY principle here.

Reply:
Done, you can resolve this.
The steps are not tested but they look accurate.
Added navigation on top to prevent excessive scrolling.